### ISSUE TITLE
Create a UI to match the proposed EDTF

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#3308]
Note; there was some confusion in both #2039 and #3301, so this is clarification. This also is going to be different that what will be shown in the OL UI.

### Problem: 
Right now, I don't believe people follow the proposed EDTF mentioned in #2039 when writing in the date. This causes issues in the data dump where the formatting is inconsistent (this is just one section of the data dump):
![image](https://user-images.githubusercontent.com/55302685/78391305-f8ee4680-759a-11ea-9fbc-630a1023e569.png).

### Solution:
create a UI for the edit page to make it easier to follow the EDTF format that includes instructions (see the table in #2039):
![image](https://user-images.githubusercontent.com/55302685/78392236-c0e80300-759c-11ea-9b96-5e17ae80a3be.png)

### Stakeholders
Anyone from both #2039 and #3301 and those who look at the data dump and those who would like to see the EDTF format be used.

**Comment by BrittanyBunk** [comment]
@tfmorris What someone inputs gets converted to the EDTF from what I see - like the table Drini created in #2039. People will enter in values - that's column 2. What it gets converted to, the EDTF, is column 3.

**Comment by guyjeangilles** [comment]
I don't think user input should be restricted to just date, not datetime. If we're trying to make the date easier to comply with EDTF, we should eliminate free forn text entry

**Comment by BrittanyBunk** [comment]
@guyjeangilles I agree. I don't think time should be required. I put in free form, due to the extra parts in the EDTF, like: 
- Approximate date | 1973? | 1973?
- Year range, both years known | 2014-2017 | 2014/2017
I'm thinking about adding in 'approximate', as that's the only thing missing. Along with your idea, this is what it'd look like:
![image](https://user-images.githubusercontent.com/55302685/78694526-a326e080-78b1-11ea-8991-9982bee655fb.png)

**Comment by BrittanyBunk** [comment]
As [per discussion](https://github.com/internetarchive/openlibrary/issues/3301#issuecomment-610538724) with @seabelis, here's an adjustment to the proposed layout: 
![image](https://user-images.githubusercontent.com/55302685/78745462-1a438f80-7919-11ea-8e06-8c8a5672e94b.png)

**Comment by BrittanyBunk** [comment]
@seabelis your thoughts?

**Comment by seabelis** [comment]
The overwhelming majority of editions have just a year or year and month. This seems to overcomplicate matters. If there has to be an input form like this, then maybe it's possible to show just the year and month fields (maybe day) with an option to extend to an 'advanced' form with the date range and BCE.

I'd really prefer to just put additional instructions on the form asking folks to use the yyyy-mm-dd format and to click for more info about advanced options.

**Comment by BrittanyBunk** [comment]
This follows the EDTF format, which it's why it's this way. You're referring to the last image I posted right? A year range I feel is relevant, but it's up to you on that.
